src/errors/AssignmentToCodeableReferenceError.ts

```diff
@@ -0,0 +1,15 @@
+import { Annotated } from './Annotated';
+import { AssignmentValueType } from '../fshtypes/rules';
+
+export class AssignmentToCodeableReferenceError extends Error implements Annotated {
+  specReferences = ['https://build.fhir.org/references.html#CodeableReference'];
+  constructor(
+    public valueType: string,
+    public value: AssignmentValueType,
+    public childPath: string
+  ) {
+    super(
+      `Cannot assign ${valueType} value: ${value.toString()} to CodeableReference. Assign to CodeableReference.${childPath} instead`
```

Comment:
```suggestion
      `Cannot assign ${valueType} value: ${value.toString()} to CodeableReference. Assign to CodeableReference.${childPath} instead.`
```
Since we already have a sentence (with punctuation) in the message, I think it makes send to have a period at the end.